app/models/comment.rb

```diff
@@ -0,0 +1,6 @@
+class Comment < ActiveRecord::Base
+  belongs_to :commentable, polymorphic: true
+  belongs_to :user
+
+  validates_presence_of :user_id, :body
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.